src/createServer.js

```diff
@@ -1,9 +1,31 @@
 /* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    let newUrl = req.url;
+
+    if (req.url.slice(0, 2) === '//') {
```

Comment:
Let's move this expression  `req.url.slice(0, 2)` to variable with meaningful variable naming